### DISCUSSION TITLE
谁解释一下这个

### DISCUSSION THREAD

**Discussion by PythonSmall-Q** [XMOJ-Script-dev/XMOJ-Script#350] [General]
![wtf](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/106425289/be37aa4f-b409-439e-9211-5ce3b9fd2d86)
 @langningchen @boomzero

**Comment by boomzero** [comment]
Emm

**Comment by boomzero** [comment]
<img width="1171" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/d5b8ced1-dce8-4978-8afd-dad2c86e3536">
because of this

**Comment by langningchen** [comment]
@PythonSmall-Q 怎么里面没有你？ 🤣

**Reply by PythonSmall-Q** [reply]
我不知道啊

**Comment by langningchen** [comment]
现在我“出名”了，AI也“认识”我了 💯

**Comment by langningchen** [comment]
AI 太棒了，给我编出了很多不存在的奖项

**Comment by boomzero** [comment]
actually, I think python is a stupid programming language

**Comment by boomzero** [comment]
ad: https://docs2.xmoj-bbs.tech
